components/monitoring/logging/base/install-logging-operator.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat 
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-operators-redhat
+  namespace: openshift-operators-redhat 
+spec: {}
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "elasticsearch-operator"
+  namespace: "openshift-operators-redhat" 
+spec:
+  channel: "stable-5.5" 
+  installPlanApproval: "Automatic" 
+  source: "redhat-operators" 
+  sourceNamespace: "openshift-marketplace"
+  name: "elasticsearch-operator"
```

Comment:
why do we need elastic search? we forward the logs to cloudwatch https://docs.openshift.com/container-platform/4.12/logging/config/cluster-logging-collector.html#cluster-logging-removing-unused-components-if-no-elasticsearch_cluster-logging-collector

Reply:
We don't need elasticsearch.

Reply:
we need this for deploying the fluentd collectors. we are not running the elasticsearch pods. But fluentd is getting deployed as elasticsearch log colletctor.

Reply:
we are using the above documentation link. thats why we are deploying only fluentd in [configure-logging.yaml](https://github.com/redhat-appstudio/infra-deployments/pull/1384/files#diff-7ba5c34961c5175a02d801bc5b8d296c62cb25ff83cbb06c76e6493505ae696e)

Reply:
let me see if we can get collectors working without the elasticsearch operator.